examples/confluent-server-oauth/README.md

```diff
@@ -0,0 +1,94 @@
+# OAuth in Kafka and Metadata Service (RBAC Server)
+
+## Overview
+
+OAuth is an open standard to obtain limited access to an account, such as Keycloak, or Okta, without
+exposing user credentials.
+This Docker Compose setup provides a container-based experience for getting started with Confluent
+Platform OAuth.
+This creates a minimal CP setup (consisting only of the broker and MDS) operating seamlessly *
+*without** LDAP.
+
+## Getting Started
+
+To run the Docker Compose setup, execute the following command:
+
+```bash
+docker compose up -d
+```
+
+This will:
+
+1. Start a Keycloak server on port 8080. This acts as the OAuth identity provider.
+2. Run Confluent Server with Role-based access control (RBAC).
+3. Create a new listener `EXTERNAL` for SASL/OAuthBearer in Confluent Server.
+4. Configure Confluent Server to connect to local Keycloak identity provider.
+
+## RBAC using OAuth
+
+1. Get the Kafka cluster ID from docker-compose file or broker logs or following REST
+   API : http://localhost:8091/v1/metadata/id
+
+2. Get an access token from Keycloak passing the client credentials of super user client app:
```

Comment:
where do those `c3Vw...` credentials come from? will they always be the same? Maybe worth an explanation of where they come from, or instructions on how to get them if they'll be different each time this is run

Reply:
Yeah, this will always be the same as we are using local keycloak identity provider with the creds hardcoded in realms-export.json file. Will include this information.